tests/test_oml/test_functional/test_metrics/test_retrieval_metrics.py

```diff
@@ -44,6 +45,42 @@ def naive_precision(positions: TPositions, k: int) -> torch.Tensor:
     return metric
 
 
+@pytest.fixture()
+def distances() -> torch.Tensor:
```

Comment:
There is a problem in the current design: `test_calc_fnmr_at_fmr` pretend that it can process some distances or masks, but the values inside it are hardcoded for some particular inputs. So, it's not consistent. 

One of the solutions: let's put (dist, gt, ignore, fmr_vals, exprected_metric) into one fixture. Then we pass it to the test function, which calls the function and checks if the expected metric and the actual one are the same. 

If you have something different in mind it's also fine if it solves the problem